tests/pages/base.py

```diff
@@ -73,15 +73,15 @@ class Navigation(BaseRegion):
         _developer_edition_page_locator = (By.CSS_SELECTOR, '.c-menu-item-link[data-link-name="Firefox Developer Edition"]')
         _manifesto_page_locator = (By.CSS_SELECTOR, '.c-menu-item-link[data-link-name="Mozilla Manifesto"]')
         _firefox_download_button_locator = (By.CSS_SELECTOR, '#protocol-nav-download-firefox > .download-link')
-        _firefox_account_button_locator = (By.CSS_SELECTOR, '.c-navigation-fxa-cta-container > .js-fxa-cta-link')
+        _mozilla_vpn_button_locator = (By.CSS_SELECTOR, '[data-cta-text="Get Mozilla VPN"]')
```

Comment:
I kind of liked that this could double as an attribution test, checking we have a useful name for the click. Happy to switch to a class name selector though.